src/lib/Libpython/pbs_python_external.c

```diff
@@ -724,7 +760,7 @@ pbs_python_run_code_in_namespace(struct python_interpreter_data *interp_data,
 
 	PyErr_Clear(); /* clear any exceptions before starting code */
 	/* precompile strings of code to bytecode objects */
-	(void) PyEval_EvalCode((PyCodeObject *)py_script->py_code_obj,
+	(void) PyEval_EvalCode((PyObject *)py_script->py_code_obj,
```

Comment:
PyEval_EvalCode() returns a new reference, don't we have to catch that and free that by PyCLEAR() or Py_XDECREF()?
https://docs.python.org/3.6/c-api/veryhigh.html#c.PyEval_EvalCode